### ISSUE TITLE
Training: allow lh and rh of a subject to be one in training, one in test set?

### ISSUE THREAD

**Issue by dfsp-spirit** [dfsp-spirit/meshlearn#6]
Currently it is possible that, of the 2 hemis of a subject, 1 is used in the training set, while the other is in the test set.

We should think about whether we want to allow that or not.

**Comment by dfsp-spirit** [comment]
This should be handled by using an input pipeline on a suitable input dir (one for training, one for test/eval data).
